extensions/smallrye-graphql/deployment/pom.xml

```diff
@@ -48,6 +48,11 @@
             <groupId>io.smallrye</groupId>
             <artifactId>smallrye-graphql-ui-graphiql</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-context-propagation-deployment</artifactId>
```

Comment:
So context propagation is mandatory for GraphQL now?

Reply:
Yes. For batch to work, and for our own internal Context to work, we need this.